misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/ZonedDateTimeFeatureAssertionsSpec.kt

```diff
@@ -161,4 +169,38 @@ abstract class ZonedDateTimeFeatureAssertionsSpec(
             }
         }
     }
+
+    describeFun("val ${dayFeature.name}") {
+        val dayVal = dayFeature.lambda
+
+        context("LocalDate with day of month 15") {
+            it("toBe(15) holds") {
+                fluent.dayVal().toBe(15)
+            }
+            it("toBe(20) fails") {
+                expect {
+                    fluent.dayVal().toBe(20)
+                }.toThrow<AssertionError> {
+                    messageContains("expect: 2009-05-15T11:24:56.831+05:30" )
```

Comment:
@robstoll Here, I am unable to change the message to $dayDescr: 15. Please let me know what I am missing.

Reply:
I am not sure what does not work for you, the following works for me:
```
messageContains("$dayDescr: 15", "$toBeDescr: 20")
```